modular-docs-manual/content/topics/module_guidelines-procedure.adoc

```diff
@@ -32,12 +32,14 @@ The procedure consists of one or more steps required to complete the procedure.
 
 For single-step procedures, use an unnumbered bullet instead of a numbered list.
 
+NOTE: Not all numbered lists in documentation are procedures. You can also use numbered lists in any module type for non-procedural sequences, such as a process flow of system actions.
+
 [discrete]
-== Procedure verification steps
+== Procedure Verification
 This section is optional. Provide the user with one or more steps to verify that the procedure provided the intended outcome. This may consist of:
 
-- An example of expected command output or 'pop-up' window the user should receive when the procedure is successful.
-- An 'action' (or 'actions') for the user, such as running a command, to determine the success or failure of the procedure.
+* An example of expected command output or pop-up window the user should receive when the procedure is successful
```

Comment:
List items with complete sentences should end with periods. See _The IBM Style Guide_ pg. 62.
```suggestion
* An example of expected command output or a pop-up window the user should receive when the procedure is successful.
```

Reply:
@rkratky For the headline case, I already explained in my comment on it that I intentionally made it headline style against the compliance for the sake of being consistent with all other headings in the manual for now, and I referenced issue #133 . I'd prefer to not have a random sentence-style heading that looks like a mistake until someone can address the issue throughout.

Regarding the periods, no, those aren't complete sentences which is why I removed the periods, per IBM SG. A complete sentence must have a subject and predicate (main verb). These are all just lengthy subjects.